src/makim/cli/__init__.py

```diff
@@ -161,5 +162,34 @@ def run_app() -> None:
         raise e
 
 
-if __name__ == '__main__':
+app = typer.Typer(help="Makim CLI")
```

Comment:
this code probably could be moved to cli/pipelines.py,
please take a look into a similar implementation for cron/scheduler: 

https://github.com/osl-incubator/makim/pull/126/files#diff-454932a65a23d849abab670ab6660126c4dbb868d699f9e32d40709bff8ac5a0R129